addons/website_event_track/models/event_track.py

```diff
@@ -636,3 +651,86 @@ def _get_track_suggestions(self, restrict_domain=None, limit=None):
         )
 
         return track_candidates[:limit]
+
+    def _get_reminder_times_warning(self):
+        return '<strong><u>Note</u></strong>: The start and end times of the talk were not specified when you asked to add them to\
+            your calendar, therefore the times indicated in this reminder correspond to those of the event.' if not self.date else ''
+
+    def _get_external_description(self):
+        """ Adding the URL of the event track into the description """
+        self.ensure_one()
+        shorten_description = shorten(html_to_inner_content(self.description), 1900)
+        event_track_url = werkzeug.urls.url_join(self.get_base_url(), self.website_url)
+
+        return f'<a href="{event_track_url}">{self.name}</a>\n{shorten_description}\n\n{self._get_reminder_times_warning()}'
+
+    def _get_event_track_reminder_dates(self):
+        if self.date:
+            date_begin = self.date
+            date_end = self.date_end
+        else:
+            date_begin = self.event_id.date_begin
+            date_end = self.event_id.date_end
+        return {'date_begin': date_begin, 'date_end': date_end}
```

Comment:
To reduce the diff a bit ?
```suggestion
        return {
            'date_begin': self.date or self.event_id.date_begin,
            'date_end': self.date_end or self.event_id.date_end,
        }
```